core/runtime/Cargo.toml

```diff
@@ -30,3 +32,10 @@ all-features = true
 [features]
 default = ["all"]
 all = ["url"]
+wpt-tests = []
+# This feature is used to disable compiling the WPT tests in the CI
+# when using `--all-features`. If this feature is enabled, we disable
+# the compilation of the WPT tests (since they will panic).
+# Clippy will be disabled for those methods as well, but they're one
+# line long.
+wpt-tests-do-not-use = []
```

Comment:
Hmmm, I think it's slightly weird to have tests be enabled by features instead of by only the `test` cfg. Can we just have a separate crate in the workspace for wpt tests, just like `boa_tester`? We could also potentially merge both of those in the future.